engines/python/setup/djl_python/tests/rolling_batch_test_scripts/test_rb_vllm_utils.py

```diff
@@ -0,0 +1,374 @@
+import sys
+import unittest
+import uuid
+from dataclasses import dataclass
+from typing import List, Optional, Dict, Union
+from collections import OrderedDict
+from unittest import mock
+from unittest.mock import Mock, MagicMock
+
+from transformers import AutoTokenizer
+
+import djl_python
+from djl_python.output_formatter import _json_output_formatter
+from djl_python.request import Request
+from djl_python.request_io import TextGenerationOutput, TextInput, Sequence, Token
+'''These Mock classes are in compliance with vllm RequestOutput version 0.4.2'''
+
+
+@dataclass
+class MockLogprob:
```

Comment:
this mock class is good. You can also try to use the fake_rolling_batcher to do some of the work like this.

Reply:
Yes, sure, we cam add this.